iOS/Readme.md

```diff
@@ -6,96 +6,112 @@ This document contains information about the Sport Buff iOS SDK setup, features
- 	3. Create a new "Run Script Phase" in your project’s target "Build Phases" and add the following snippet:
 
-```bash
-“${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/BuffSDK.framework/strip-frameworks.sh"
-```
+### 1. CocoaPods
 
-In Xcode 11.x you might get the following error:
+[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website. To integrate SportBuff into your Xcode project using CocoaPods, specify it in your Podfile:
 
-```swift
-dyld: Library not loaded: @rpath/BuffSDK.framework/BuffSDK
-Reason: image not found
-```
+````
+pod 'SportBuff', '0.0.34.0'
+````
+<br>
 
-If that's the case, do the following steps:
+### 2. XCFramework
 
-- Under Build Phases, click on + icon to add a new phase, and select New Copy Files Phase
-- Drag the newly created Copy Files phase above Compile Sources phase
-- In the new Copy Files phase, select Frameworks from Destination dropdown
-- Leave the subpath blank. Let by default Copy only when installing
-- Under the table, click '+' and then select BuffSDK.framework
-- Make sure Code Sign on Copy is checked (ticked)
-- Do a clean build.
+To integrate the SDK as a xcframework, you will need to follow the following steps:
+ 	1. Download the SportBuff.xcframework
+ 	2. Drag & drop SportBuff.xcframework to your project's target "Frameworks, Libraries, and Embedded Content" section under the "General" tab, and make sure that the "Embed & Sign" is selected
+
+![img](framework.png)
+<br><br>
 
 ## **Using the SDK**
 
-1. Import the BuffSDK framework header in your app delegate
+1. Import the SportBuff framework header in your app delegate
 
    ```swift
-   import BuffSDK
+   import SportBuff
    ```
 
 2. Authenticate your SDK and user:
    The SDK requires a JWT Token, in order to authenticate. In order to obtain that token, you have two options
-
+   
+<br>
 #### *Option A*
 
+Authenticate the user using our Rest API, in your own backend and then sign in the user with the following method:
+
+```swift
+SportBuff.initialize(userToken: "<UserToken>")
+```
+
+##### userToken: 
+
+**Required** The user JWT Token you've received from our Backend API service
+
+*The procedure is explained in the API Documentation section*
+
+<br>
+#### *Option B*
+
+
 Authenticate directly in the application using the SDK Key/Secret pair you did receive from us:
 
 ```swift
-BuffSDK.instance.start(key: <#SDK KEY#>, secret: <#SDK SECRET#>, userId: <#Unique User Id#>, email: <#User Email#>, photoUrl: <#User Profile Picture URL#>)
+SportBuff.initialize(sdkKey: "<SDK Key>",
+                     sdkSecret: "<SDK Secret>",
+                     uuid: "<UUID>",
+                     username: "<Username>",
+                     firstName: "<First Name>",
+                     lastName: "<Last Name>",
+                     email: "<Email>",
+                     locale: "<Locale>") { userToken in
+}
 ```
```

Comment:
We should also add that option B is only allowed to be used in Development/Testing mode only. 
Option A is to be used in Production